packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -402,12 +401,14 @@ abstract class ImageProvider<T> {
             };
             return true;
           }());
-          FlutterError.onError(FlutterErrorDetails(
-            context: ErrorDescription('while checking the cache location of an image'),
-            informationCollector: collector,
-            exception: exception,
-            stack: stack,
-          ));
+          if (FlutterError.onError != null) {
+            FlutterError.onError!(FlutterErrorDetails(
```

Comment:
More of a question: is `FlutterError.onError?(....)` allowed?

Reply:
No but you can use `FlutterError.onError?.call(....)`